backend/uclapi/dashboard/webhook_views.py

```diff
@@ -0,0 +1,187 @@
+from django.http import JsonResponse
```

Comment:
Prefer to use PrettyJsonResponse here. You can import Jaro's prettifier as JsonResponse to avoid rewriting code below. It's easier for debugging / viewing in a browser 👍